.github/workflows/test.yml

```diff
@@ -15,37 +15,84 @@ jobs:
       - name: Configure 1Password Connect
         uses: ./configure # 1password/load-secrets-action/configure@<version>
         with:
-          connect-host: http://localhost:8080
+          connect-host:	http://localhost:8080
           connect-token: ${{ secrets.OP_CONNECT_TOKEN }}
       - name: Load secrets
+        id: load_secrets
         uses: ./ # 1password/load-secrets-action@<version>
         env:
-          SECRET: op://v5pz6venw4roosmkzdq2nhpv6u/hrgkzhrlvscomepxlgafb2m3ca/password
-          SECRET_IN_SECTION: op://v5pz6venw4roosmkzdq2nhpv6u/hrgkzhrlvscomepxlgafb2m3ca/test-section/password
-          UNMASKED_VALUE: op://v5pz6venw4roosmkzdq2nhpv6u/hrgkzhrlvscomepxlgafb2m3ca/test-section/username
```

Comment:
I see that the tests where the references have IDs have been removed. Any reason for doing so?
As a dev, this would give me confidence that both options work well with the action.

Let me know if there's anything you need to bring back these tests (if you feel like it makes sense to include them).

Reply:
I tried to use with those ids, but test fails with the error ```status 403: Authorization: token does not have access to vault v5pz6venw4roosmkzdq2nhpv6u```